docs/docs/models-usage.md

```diff
@@ -534,6 +534,19 @@ User.findAll({
   */
 })
 ```
+This will produce an outer join. However, a `where` clause on a related model will create an inner join and return only the instances that have matching sub-models. To return all the instances, you should add `required: false`.
```

Comment:
Not sure it makes sense to add this under nested eager laoding - it should probably be higher up